packages/ui-components/src/features/Auth/AuthViewWrapper.tsx

```diff
@@ -0,0 +1,45 @@
+/*
+ * Tupaia
+ *  Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+
+import React, { ReactNode } from 'react';
+import styled from 'styled-components';
+import { Paper, Typography } from '@material-ui/core';
+
+const Wrapper = styled(Paper)`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  max-width: 100%;
+  box-shadow: none;
+  padding: 3rem 0;
```

Comment:
I think there should be some left and right padding so that messages and error messages have some spacing on the sides